test-docker-compose.yaml

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2023 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+version: '3'
+services:
+
+  api:
+    container_name: 'kernelci-api'
```

Comment:
I think it'd be better to use different container names than the ones used for production to make it clear they're just for testing and can be safely destroyed after running the tests. E.g. you can use something like a `test-` prefix for a name.

Reply:
Yes, we need to rename the containers.
Also, I will rename the mongodb volume used for tests to avoid accessing data from `db` service instead.